extensions/mini/__init__.py

```diff
@@ -97,3 +102,22 @@ def register(self):
             await message.channel.send(embed=embed)
         else:
             await message.channel.send("Role not found")
+
+    async def avatar(self, args, message):
+        parser = argparse.ArgumentParser(description=self.bot.processOutput("Shows an clickable avatar of a user", message))
+        parser.add_argument("avatar", nargs="?", default=str(message.author.id), help=self.bot.processOutput("The user of the avatar you want to get", message))
+        args = parser.parse_args(args)
+        avatar = message.channel.guild.get_member(self.bot.getIDFromTag(args.avatar))
+        if avatar != None:
+            embed = discord.Embed(
+                color = discord.Colour.from_rgb(r=255,g=192,b=60)

```

Comment:
Use the user's color instead of the Synixe yellow. See line 77 in ?card for an example